lib/rules/require-shorthand-arrow-functions.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Require shorthand arrow function expressions when returning a single statement.
```

Comment:
Yeah since arrow functions are just a shorter syntax for function expressions. This is copied from airbnb styleguide. Although that isn't exactly the point of the example so 

```js
// expression body (single expression so don't need curly braces and can do implicit return)
nums.map(v => v + 1);
// statement body (two statement requires curly braces + return statement
nums.map(v => {
  v = v + 1;
  return v;
});
```